plugins/woocommerce/tests/e2e-pw/utils/editor.js

```diff
@@ -45,6 +45,7 @@ const goToPageEditor = async ( { page } ) => {
 	);
 	await page.goto( 'wp-admin/post-new.php?post_type=page' );
 	await disableWelcomeModal( { page } );
+	await closeChoosePatternModal( { page } );
```

Comment:
There are many tests with closing pattern needed. Let's keep this inside the editor util.